frontend/packages/core/src/Input/date-time.tsx

```diff
@@ -26,8 +26,9 @@ const DateTimePicker = ({ onChange, ...props }: DateTimePickerProps) => (
     <MuiDateTimePicker
       renderInput={inputProps => <PaddedTextField {...inputProps} />}
       onChange={(value: Dayjs | null) => {
-        if (value && value.isValid()) {
-          onChange(value.toDate());
+        if (!value || value.isValid()) {
```

Comment:
Should we do this in the time picker as well?

Reply:
i don't have a scenario for the time picker where we need it but I can certainly do it too